contracts/FreakTeam.sol

```diff
@@ -0,0 +1,66 @@
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/token/ERC1155/ERC1155.sol";
+import "hardhat/console.sol";
+import "@openzeppelin/contracts/access/AccessControl.sol";
+
+
+
+enum Role{it, supportIt, intern}
+enum Skill{novice, advanced, competent, profiency, expert}
+
+struct Freak {
+        string name;
+        uint256 startDate;
+        uint256 employeeNumber;
+        Role choiceRole;
+        Skill choiceSkill;
+
+}
+
+
+contract FreakTeam is ERC1155, AccessControl {
+
+    mapping(address => Freak)  public freaks;
+    address[] public freakAccts;
+
+    bytes32 public constant FINANCIAL_ROLE = keccak256("FINANCIAL");
```

Comment:
There seems to be an inconsistent style in naming the roles: one is `FINANCIAL` and the others are ended with `ROLE`